metisp/pymetis/src/pymetis/base/recipe.py

```diff
@@ -27,28 +27,34 @@ class MetisRecipe(cpl.ui.PyRecipe):
         The abstract base class for all METIS recipes.
         In an ideal world it would also be abstract (derived from ABC, or metaclass=abc.ABCMeta),
         but `pyesorex` wants to instantiate all recipes it finds
-        and would crash with an abstract class.
+        and would crash if it were an abstract class.
+
         The underscored _fields must be present but should be overwritten
-        by every child class (`pyesorex` actually checks for their presence).
+        by every child class (and `pyesorex` actually checks for their presence).
     """
     _name = "metis_abstract_base"
     _version = "0.0.1"
     _author = "METIS PIP team"
-    _email = "astar.vienna@univie.ac.at" # ToDo some sensible default maybe?
-    _copyright = "CPL-3.0-or-later"
+    _email = "astar.vienna@univie.ac.at"                        # ToDo is this a sensible default?
+    _copyright = "GPL-3.0-or-later"                             # I guess we're using the same copyright everywhere
     _synopsis = "Abstract-like base class for METIS recipes"
-    _description = "This class serves as the base class for all METIS recipes."
+    _description = ("This class serves as the base class for all METIS recipes."
+                    "Bonus points if it is not visible from pyesorex.")
 
-    parameters = cpl.ui.ParameterList([])          # By default, classes do not have any parameters
-    implementation_class: type = str               # Dummy class, this must be overridden in the derived classes anyway
+    parameters = cpl.ui.ParameterList([])           # By default, a recipe does not have any parameters.
```

Comment:
Maybe add a dummy parameter here? E.g. (untested)
```suggestion
    # By default, a recipe does not have any parameters.
    # However, a default dummy parameter is added because
    # recipes without parameters crash in esorex. See
    # https://jira.eso.org/browse/PIPE-12044
    parameters = cpl.ui.ParameterList([
        cpl.ui.ParameterEnum(
            name="dummy",
            context="dummy",
            description="dummy",
            default="dummy",
            alternatives=("dummy"),
        ),
    ])
```

Reply:
This will make other tests fail: I verify that the `context` of the parameter is the same as the name of the recipe. But maybe that can be relaxed / deleted.

Reply:
Ah, that's a useful test. Thanks for fixing it like this. A bit ugly, but at least it is clear